src/action/index.js

```diff
@@ -15,6 +16,9 @@ when(
       nav.reset('PinCheck');
       return;
     }
+    if (Platform.OS === 'android') {
+      await backup.authenticate();
+    }
```

Comment:
This check seems redundant as there is already a check in `cloudStore.js`

```suggestion
    await backup.authenticate();
```